### ISSUE TITLE
Clicking into a grant and backing out within the advocacy collection causes only one grant to show up

### ISSUE THREAD

**Issue by nfrgosselin** [gitcoinco/web#10022]

**Comment by nfrgosselin** [comment]
@Jer-Sch This also doesn't seem to be working as the ticket indicates. Right now when I click into a grant's detail page from a collection, the detail page opens in a new tab (so the back button is disabled) and the "Back to grants" link clicks to the main Grants Explorer.

I actually think the UX I described above is fine, but wanted to make sure that was intended

**Comment by Jer-Sch** [comment]
@nfrgosselin Yeah, I think that the acceptance criteria was not updated on these tickets. Aditya, Chibie, Kevin, and I had discussed multiple approaches to fixing this, and ended up settling on just opening the details page in a new tab. But that was intended to be a quick-fix and not a long term solution. I think creating stories for that long term solution then fell off the radar.

**Comment by nfrgosselin** [comment]
Ahh okay makes sense — thanks for clarifying. I just updated the criteria —> this looks good to me!

**Comment by Jer-Sch** [comment]
Great thanks! Sorry for the outdated information -- that was my fault

**Comment by EmHsiehGitcoin** [comment]
If I'm clicking from the title link that works for me on every grant aka opens up a new window. However, not sure why there are cards that has "View Grant" CTA instead of "+Add to Cart". Idle grants, maybe? These cards don't open new windows. For example, "owocki forever!" collection has three of those and they consistently behaving this way. 

Interestingly, the _maybe-idle_ grants in "Pixelant" collection triggered the original problem: 
1/ when I click "View Grant" CTA from one of the grants  in Pixelant collection
2/ It opens the grant detail page in the same tag (i.e. no new tab opened)
3/ When I click "Back to Collections" or back button 
4/ I'm back to "Pixelant's Collection" page but only seeing one grant instead of the 6 grants I saw at first

Even more interesting, the same grant would behave differently in "Pixelant" versus in "owocki forever!" collection.
- owocki forever! _maybe-idle_ grants w/ "View Grant" CTA: don't open new window but they work normal when I click **"back to collections"** 
- pixelant _maybe-idle_ grants w/ "View Grant" CTA: don't open new window + only see one grant when I click **"back to Grants"** 
(notice they have different copy on text link as well)

-------------------------------------------------

I think the acceptance criteria should be:

GIVEN I am viewing grants in a collection 
WHEN I click on a grant from the grant title or View Grant CTA
THEN the grant's detail page opens in the same tab
AND I see the back text link says "Back to {collection name}"
WHEN I click on "Back to {collection name}"
THEN I am back to the collection

**Comment by Jer-Sch** [comment]
@EmHsiehGitcoin Huh, that is interesting 🤔 I'd be happy to take a look at this on Monday.

**Comment by Jer-Sch** [comment]
@EmHsiehGitcoin @nfrgosselin So it looks like active grants show "Add to Cart" and inactive grants (not updated in the past three months) show "View Grant". I'm not sure at this point why the back button works fine for owocki forever! grants but not for pixelant grants.

**Comment by Jer-Sch** [comment]
@EmHsiehGitcoin @nfrgosselin For now, do we want to open a new tab when "View Grant" is clicked?

**Comment by gdixon** [comment]
> @EmHsiehGitcoin @nfrgosselin So it looks like active grants show "Add to Cart" and inactive grants (not updated in the past three months) show "View Grant". I'm not sure at this point why the back button works fine for owocki forever! grants but not for pixelant grants.

I believe this is due to the pagination we have in place - the link you are sent back to from the details page is: 

.../grants/explorer/?**page=2**&limit=12&me=false&sort_option=weighted_shuffle&collection_id=3...

But there are only 6 items in the collection so scrolling to the bottom of that page shouldn't have incremented the `page` number (as each page should hold 12 items).
